src/simtools/model/model_parameter.py

```diff
@@ -421,29 +373,24 @@ def change_parameter(self, par_name, value):
         InvalidModelParameterError
             If the parameter to be changed does not exist in this model.
         """
-        if par_name not in self._parameters:
-            msg = f"Parameter {par_name} not in the model"
-            self._logger.error(msg)
-            raise InvalidModelParameterError(msg)
+        if par_name not in self.parameters:
+            raise InvalidModelParameterError(f"Parameter {par_name} not in the model")
 
-        if isinstance(value, str):
-            value = gen.convert_string_to_list(value)
+        value = gen.convert_string_to_list(value) if isinstance(value, str) else value
 
+        param_type = self.get_parameter_type(par_name)
```

Comment:
Not crazy about `param_type` and `par_name` being inconsistent abbreviations.

Reply:
Fixed.